usaspending_api/database_scripts/etl/award_delta_view.sql

```diff
@@ -1,6 +1,65 @@
+        w.submission_fiscal_year,
+        w.submission_fiscal_month
+    from dabs_submission_window_schedule w
+    where
+    	-- For COVID, look only after a certain date (likely on/after 2020-04-01, or the start of FY2020 FYP07)
+        w.period_start_date >= '2020-04-01' -- using older date to see data surface
+        and w.submission_reveal_date <= now() -- change "period end" with "window close"
+    order by w.submission_fiscal_year desc, w.submission_fiscal_month desc
+),
+    eligible_submissions AS (
+        SELECT *
+        FROM submission_attributes s
+        WHERE concat(s.reporting_fiscal_year::text, lpad(s.reporting_fiscal_period::text, 2, '0')) IN (
+            SELECT fyp FROM closed_periods
+        )
+    ),
+    eligible_file_c_records AS (
+        SELECT
+            faba.award_id,
+            CASE WHEN defc.group_name = ('covid_19') THEN faba.gross_outlay_amount_by_award_cpe else 0 END AS gross_outlay_amount_by_award_cpe,
+            CASE WHEN defc.group_name = ('covid_19') THEN faba.transaction_obligated_amount else 0 END AS transaction_obligated_amount,
+            faba.disaster_emergency_fund_code,
+            s.reporting_fiscal_year,
+            s.reporting_fiscal_period
+        FROM financial_accounts_by_awards faba
+        INNER JOIN eligible_submissions s ON s.submission_id = faba.submission_id
+        INNER JOIN disaster_emergency_fund_code defc on defc.code = faba.disaster_emergency_fund_code
+        WHERE faba.disaster_emergency_fund_code IS NOT NULL
+    ),
+    fy_final_outlay_balances AS (
+        -- Rule: If a balance is not zero at the end of the year, it must be reported in the
+        -- final period's submission (month or quarter), otherwise assume it to be zero
+        SELECT faba.award_id, sum(faba.gross_outlay_amount_by_award_cpe) AS prior_fys_outlay,
+            faba.disaster_emergency_fund_code
+        FROM eligible_file_c_records faba
+        GROUP BY
+            faba.award_id,
+            faba.reporting_fiscal_period,
+            faba.disaster_emergency_fund_code
+        HAVING faba.reporting_fiscal_period = 12
+        AND sum(faba.gross_outlay_amount_by_award_cpe) > 0
+    ),
+     current_fy_outlay_balance AS (
+        SELECT
+            faba.award_id,
+            faba.reporting_fiscal_year,
+            faba.reporting_fiscal_period,
+            faba.disaster_emergency_fund_code,
+            sum(faba.gross_outlay_amount_by_award_cpe) AS current_fy_outlay
+        FROM eligible_file_c_records faba
+        GROUP BY
+            faba.award_id,
+            faba.reporting_fiscal_year,
+            faba.reporting_fiscal_period,
+            faba.disaster_emergency_fund_code
+        HAVING concat(faba.reporting_fiscal_year::text, lpad(faba.reporting_fiscal_period::text, 2, '0')) IN
+        (SELECT max(fyp) FROM closed_periods) AND sum(faba.gross_outlay_amount_by_award_cpe) > 0)
```

Comment:
Again the `having` is better suited as a `where`. See updated SQL attached to https://federal-spending-transparency.atlassian.net/browse/DEV-5283